requirements.txt

```diff
@@ -2,7 +2,7 @@
 -r requirements_production.txt
 
 # Requirements for development and tests in alphabetical order
-coverage
+coverage>=4.0a5,<4.1.0
```

Comment:
Yes you are right. We shouldn't use development version.

Here in this case I would apply for a special exception. Coverage 4.0 will now support the use of setup.cfg and I prefer this. The latest alpha release was on 17th Feb. 2015 and the [development](https://bitbucket.org/ned/coveragepy/commits/all) goes straight ahead. When coverage 4.0 final is release I could change this file immendiately.